model/common/tests/stencil_tests/test_mo_intp_rbf_rbf_vec_interpol_vertex.py

```diff
@@ -40,9 +43,10 @@ def reference(
         return dict(p_v_out=p_v_out, p_u_out=p_u_out)
 
     @pytest.fixture
-    def input_data(self, grid):
-        if grid.get_offset_provider("V2E").has_skip_values:
+    def input_data(self, grid: base.BaseGrid) -> dict:
+        if grid.get_offset_provider("C2E2C2E").has_skip_values:
             pytest.xfail("Stencil does not support missing neighbors.")
+
```

Comment:
Should use the corresponding marker. Skip this comment if you plan to fix it directly in the next PR.